qadence/backends/pyqtorch/convert_ops.py

```diff
@@ -245,13 +245,26 @@ def convert_block(
             generator = convert_block(block.generator, n_qubits, config)[0]  # type: ignore[arg-type]
         time_param = config.get_param_name(block)[0]
 
+        # convert noise operators here
+        noise_operators: list = [
+            convert_block(noise_block, config=config)[0] for noise_block in block.noise_operators
+        ]
+        if len(noise_operators) > 0:
+            # squeeze batch size for noise operators
+            noise_operators = [
+                pyq_op.tensor(full_support=qubit_support).squeeze(-1) for pyq_op in noise_operators
+            ]
```

Comment:
Do I understand correctly that this is blowing up the qubit support of each noise operator to the full qubit support of the evolution?

If yes, is this a current limitation of the `mesolve` ?

Reply:
@jpmoutinho Yes, it is a current limitation of mesolve.